WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -133,6 +134,17 @@ private void StepConnectionConfirmationPhase()
 					var removedAliceCount = round.Alices.RemoveAll(x => alicesDidntConfirm.Contains(x));
 					round.LogInfo($"{removedAliceCount} alices removed because they didn't confirm.");
 
+					if (round.InputCount >= Config.MinInputCountByRound)
+					{
+						await foreach (var offendingAlices in CheckTxoSpendStatusAsync(round, cancel).ConfigureAwait(false))
+						{
+							if (offendingAlices.Any())
+							{
+								var removed = round.Alices.RemoveAll(x => offendingAlices.Contains(x));
+								round.LogInfo($"There were {removed} alices removed because they spent the registered UTXO.");
+							}
+						}
+					}
```

Comment:
I would put a blank line after this line. I have mistakingly though that this is an `else` branch for L137.